backend/tests/unit/message_bus/operations/requests/test_proposed_change.py

```diff
@@ -128,8 +128,8 @@ async def test_get_proposed_change_schema_integrity_constraints(
     )
     non_generate_profile_constraints = [c for c in constraints if c.constraint_name != "node.generate_profile.update"]
     # should be updated/removed when ConstraintValidatorDeterminer is updated (#2592)
-    assert len(constraints) == 82
-    assert len(non_generate_profile_constraints) == 17
+    assert len(constraints) == 127
```

Comment:
@ajtmccarty Please can you check this one too, I'm not 100% sure this is expected

Reply:
this is expected. these two numbers are mostly a reminder that we could do a better job determining which constraints actually need to run. in this case, we probably don't need to run the new inherit_from checker unless the `inherit_from` value on a node has actually changed, but the constraint checker infrastructure does not have a way to handle that kind of logic right now